spec/model/associations_spec.rb

```diff
@@ -8,10 +8,21 @@
 
     context "single has_many association" do
       before { Foo::User.has_many :comments }
+      subject { super()[:has_many] }
```

Comment:
while we're refactoring think we should use an explicit subject instead of calling `super()`?

```ruby
context "setting associations without options" do
  subject(:associations) { User.associations }
  # ...
  context "single has_many association" do
    subject { associations[:has_many] }
  end
```